Achievements/SelfMade.lua

```diff
@@ -32,21 +31,21 @@ function self_made_achievement:Unregister()
     self:RegisterEvent("ITEM_UNLOCKED")
```

Comment:
should be self:UnregisterEvent ...

Reply:
> should be self:UnregisterEvent ...

did a bad copy pasta there and forgot to change it. Thanks for the catch